### ISSUE TITLE
Comments in data section cause np.reshape errors.

### ISSUE THREAD

**Issue by ahype-loves-geoscience** [kinverarity1/lasio#319]
las_4834.las cannot reshape array of size 9858 into shape (7)

If I remove the comments it runs to completion. 

Sample lines below. 

```
~ASCII Test Data
#	ETIM	BSG1	BQP1	POMS	POPV	HMS1
#--------------------------------------------------------
      0.300	  2355.765	  2343.880	     0.000	     0.000	     0.000	
```

Ahype

**Comment by ahype-loves-geoscience** [comment]
Problem can be solved by ignoring the commented out lines in read_data_section_iterative.

Such as. 

    def items(f):
        for line in f:
            if line.startswith('#'):
                continue

**Comment by kinverarity1** [comment]
Thanks for raising this issue! It seems like a straightforward fix, thanks.

**Comment by kinverarity1** [comment]
@ahype-loves-geoscience this error should have been fixed in the `master` branch by the PR #391 which I have just merged. Please feel free to re-open this issue if you find that that does not fix the issue for you.
